src/Compilers/VisualBasic/Portable/Symbols/Source/SourcePropertySymbol.vb

```diff
@@ -734,6 +734,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                                                                             isOverrides:=True, isWithEvents:=Me.IsWithEvents))
                 End If
 
+                Debug.Assert(IsDefinition)
```

Comment:
Scratch this. We don't have required information for asserting there, but I covered all the call sites that we have right now and I will add a doc comment.